pkg/kubelet/prober/worker.go

```diff
@@ -62,6 +63,11 @@ type worker struct {
 	resultRun int
 }
 
+const (
+	// jitter factor for worker.doProbe sync loop
+	workerProbeJitterFactor = 1.0
```

Comment:
Note in the prober. I think we only need to worry about per-pod sync loops, and I can't think of any others off the top of my head.